### PR TITLE
[Nokia-7215] Set timeout = 180s for config reload cmd

### PR DESCRIPTION
**Pull request by lizhijianrd**
### Description of PR

Summary:
For Nokia-7215, 120 seconds is not enough for `config reload` command. Extend the time to 180s to make testcase stable.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
For Nokia-7215, 120 seconds is not enough for `config reload` command. Extend the time to 180s to make testcase stable.

#### How did you do it?
Set `config reload` timeout to 180s for Nokia-7215.

#### How did you verify/test it?
Verified on Nokia-7215 Mx testbed.

### CODE REVIEW

File: tests/platform_tests/test_reload_config.py [hunk 1]
```diff
@@ -141,6 +141,11 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     @summary: This test case is to test various system checks in config reload
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    hwsku = duthost.facts["hwsku"]
+
+    config_reload_timeout = 120
+    if hwsku in ["Nokia-M0-7215", "Nokia-7215"]:
+        config_reload_timeout = 180
```
Comment (lipxu) [inline tests/platform_tests/test_reload_config.py:148]: Is it the day1 failure? or the failure was introduced by the execute_config_reload_cmd change?

**Review by lipxu — APPROVED** [review]
LGTM

File: tests/platform_tests/test_reload_config.py (hunk 1 above)
Comment (lizhijianrd) [reply tests/platform_tests/test_reload_config.py:148]: I checked the test history, it should not be introduced by testcase change.
I can see this testcase always pass before fips is enabled in image. So it looks like fips introduce more time to `config load` and test start to fail.

**Review by wangxin — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Comment by lizhijianrd** [comment]
@yxieca  Can you please help add "Approved for 202311 branch" tag? Thanks!
@bingwang-ms Can you please help add "Approved for 202405 branch" tag? Thanks!

**Comment by lizhijianrd** [comment]
> @lizhijianrd PR conflicts with 202405 branch

Manually backport: #13969

**Pull request merged**